src/class-elements/private-setter-is-not-a-own-property.case

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2019 Caio Lima (Igalia SL). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Private setter is not stored as an own property of objects
+info: |
+  PrivateFieldGet (P, O)
+    1. Assert: P is a Private Name.
+    2. If O is not an object, throw a TypeError exception.
+    3. If P.[[Kind]] is "field",
+      a. Let entry be PrivateFieldFind(P, O).
+      b. If entry is empty, throw a TypeError exception.
+      c. Return entry.[[PrivateFieldValue]].
+    4. Perform ? PrivateBrandCheck(O, P).
+    5. If P.[[Kind]] is "method",
+      a. Return P.[[Value]].
+    6. Else,
+      a. Assert: P.[[Kind]] is "accessor".
+      b. If P does not have a [[Get]] field, throw a TypeError exception.
+      c. Let getter be P.[[Get]].
+      d. Return ? Call(getter, O).
+template: default
+features: [class-methods-private]
+---*/
+
+//- elements
+set #m(v) { this._v = v; }
+
+checkPrivateSetter() {
+  assert.sameValue(Object.getOwnPropertyDescriptor(this, "#m"), undefined);
+  assert.sameValue(Reflect.getOwnPropertyDescriptor(this, "#m"), undefined);
+
+  assert.sameValue(this.hasOwnProperty("#m"), false);
+  assert.sameValue(Reflect.has(this, "#m"), false);
+
+  assert.compareArray(Object.getOwnPropertyNames(this), []);
+  assert.compareArray(Reflect.ownKeys(this), []);
+
+  assert.sameValue("#m" in this, false);
+
+  const descriptors = Object.getOwnPropertyDescriptors(this);
+  assert.sameValue("#m" in descriptors, false);
+
+  this.#m = "Test262";
+  assert.sameValue(this._v, "Test262");
```

Comment:
same observations from src/class-elements/private-getter-is-not-a-own-property.case can be applied here